.pre-commit-config.yaml

```diff
@@ -167,7 +167,7 @@ repos:
         language: docker_image
         entry: koalaman/shellcheck:stable -x -a
         types: [shell]
-        files: ^breeze$|^breeze-complete$
```

Comment:
No need for that. [shell] will find the file. It's only files without .sh extension that have to be added here.

Reply:
I don't know why my file was not detected and I had to make these changes.